### ISSUE TITLE
python scripts core dump with non-default locale 

### ISSUE THREAD

**Issue by tarzanek** [scylladb/scylla#7408]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [X] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): 4.2.rc4-0.20200907.bf0c493c28
Cluster size: 3
OS (RHEL/CentOS/Ubuntu/AWS AMI): Centos 7

*Hardware details (for performance issues)*          Delete if unneeded
Platform (physical/VM/cloud instance type/docker):
Hardware: sockets= cores= hyperthreading= memory=
Disks: (SSD/HDD, count)

```
[scylla@scylla ~]$  sudo /usr/lib/scylla/node_exporter_install --help
usage: node_exporter_install [-h] [-F]

Download and install prometheus node_exporter

optional arguments:
  -h, --help   show this help message and exit
  -F, --force  Force re-installation when node_exporter is already installed
[scylla@scylla ~]$ scylla --version
4.2.rc4-0.20200907.bf0c493c28
[scylla@scylla ~]$ locale
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC="en_US.UTF-8"
LC_TIME="en_US.UTF-8"
LC_COLLATE="en_US.UTF-8"
LC_MONETARY="en_US.UTF-8"
LC_MESSAGES="en_US.UTF-8"
LC_PAPER="en_US.UTF-8"
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT="en_US.UTF-8"
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
[scylla@scylla ~]$ export LC_ALL=en_US
[scylla@scylla ~]$ locale
LANG=en_US.UTF-8
LC_CTYPE="en_US"
LC_NUMERIC="en_US"
LC_TIME="en_US"
LC_COLLATE="en_US"
LC_MONETARY="en_US"
LC_MESSAGES="en_US"
LC_PAPER="en_US"
LC_NAME="en_US"
LC_ADDRESS="en_US"
LC_TELEPHONE="en_US"
LC_MEASUREMENT="en_US"
LC_IDENTIFICATION="en_US"
LC_ALL=en_US
[scylla@scylla ~]$  sudo /usr/lib/scylla/node_exporter_install --help
Segmentation fault
```

**Comment by tarzanek** [comment]
I suppose the same problem happens for any other relocatable python scripts

**Comment by tarzanek** [comment]
quick workaround is of course 
`export LC_ALL=en_US.UTF-8`

**Comment by avikivity** [comment]
Yes, we should just put that into the thunk.

**Comment by avikivity** [comment]
Backported to 4.0, 4.1, 4.2.

**Comment by eliransin** [comment]
Fix is part of 2020.1.4
